includes/functions.php

```diff
@@ -108,6 +108,42 @@ function tsml_bounds()
     }
 }
 
```

Comment:
This method expects an array based log of changes from `tsml_get_changed_import_meetings()` to spit out a table. That's it. Just so we don't have to cobble a table together in the meeting change detection code.

Reply:
Here's the table report generated when tsml_debug is true:
![image](https://github.com/user-attachments/assets/7e31c43f-1b02-443f-84e4-51c1a30c6249)

Here's the table report we had before: 
![image](https://github.com/user-attachments/assets/b6e503cb-5648-4a27-b8d5-733ee39b7540)
The new report has the Header text centered in each column. Think the new report might be better if left aligned and delineated by grid lines so it's easier to read.

Reply:
I'm kinda up the in air about the Notes column for starters, not sure if folks would _want_ to see the system names for the fields changed. I agree the header columns should be left aligned, will add that. I think the colors kinda throw it off, and the table border looks a little odd, even if it adds separation. I'll tweak the header cols to left align and try that.